### PR TITLE
Clarify enabled features and defined features in PackageData

### PR DESCRIPTION
**Pull request by edwin0cheng**

### CODE REVIEW

File: crates/project_model/src/cargo_workspace.rs [hunk 1]
```diff
@@ -99,7 +99,9 @@ pub struct PackageData {
     pub dependencies: Vec<PackageDependency>,
     /// Rust edition for this package
     pub edition: Edition,
-    /// List of features to activate
+    /// Features provided by the crate, mapped to the features required by that feature.
+    pub defined_features: FxHashMap<String, Vec<String>>,
+    /// List of features enabled on this package
     pub features: Vec<String>,
```
Comment (matklad) [inline crates/project_model/src/cargo_workspace.rs:105]:
```suggestion
    pub features: FxHashMap<String, Vec<String>>,
    /// List of features enabled on this package
    pub active_features: Vec<String>,
```

Comment (edwin0cheng) [reply crates/project_model/src/cargo_workspace.rs]: Fixed.

**Comment by edwin0cheng** [comment]
The new `features` is not used right now, but will be handy in future as implementing incremental crate update and proper cfg handling.

**Pull request merged**